modules/envoy-gateway/main.tf

```diff
@@ -63,15 +63,6 @@ spec:
           - op: replace
             path: /metadata/name
             value: ${var.cluster_issuer_name}
-      - target:
-          kind: SecurityPolicy
-        patch: |-
-          - op: replace
-            path: /spec/jwt/providers
-            value:
-              - name: auth0
-                remoteJWKS:
-                  uri: ${var.auth0_jwks_uri}

```

Comment:
This was causing us issues to apply a blanket SecurityPolicy at the gateway to require JWT be provided due to the nature of how the ACME challenge needs to be solved:
https://letsencrypt.org/docs/challenge-types/#http-01-challenge

Because JWT was required the traffic to solve the challenge was being rejected due to HTTP 401.

Instead, we will expect that security policies will be individually applied to all HTTP Routes that are created for the cluster - **Like what is already in place for the telemetry endpoints**: [Like what is shown in this file](https://github.com/Sage-Bionetworks-Workflows/eks-stack/blob/ibcdpe-1095-signoz-move-to-lets-encrypt/modules/signoz/resources-otel-ingress/security-policy.yaml#L1-L12)